synapse/__init__.py

```diff
@@ -21,9 +21,14 @@
 import sys
 from typing import Any, Dict
 
+from PIL import ImageFile
+
 from synapse.util.rust import check_rust_lib_up_to_date
 from synapse.util.stringutils import strtobool
 
+# Allow truncated JPEG images to be thumbnailed.
```

Comment:
What's a truncated JPEG? E.g. we only have the first say 60% of the JPEG file?

Reply:
Yup, which turns out to be relatively common for reasons beyond understanding.